src/clj/witan/send/utils.clj

```diff
@@ -14,7 +14,7 @@
             [witan.send.utils :as u])
   (:import [org.HdrHistogram IntCountsHistogram DoubleHistogram]))
 
-(set! *print-length* 20000)
+(set! *print-length* 9999999999)
```

Comment:
this is a bad place to do this as it will affect anything that compiles or uses this file